typescript/test/testdata/showcase/src/v1beta1/echo_client.ts.baseline

```diff
@@ -287,9 +284,19 @@ export class EchoClient {
   // -------------------
   // -- Service calls --
   // -------------------
-  /*
-  This method simply echos the request. This method is showcases unary rpcs. 
-  */
+/**
+ * @param {Object} request
+ *  The request object that will be sent.
+ * @param {string} request.content 
+ *  The content to be echoed by the server. 
+ * 
```

Comment:
If possible, we should not have empty lines between `@param` lines.

Reply:
Of course, just found that it's not from template, but from comments in proto files, they all have an empty line by the end of each comments of parameter.